iphone/Maps/Categories/UIColor+MapsMeColor.m

```diff
@@ -3,110 +3,9 @@
 #import "SwiftBridge.h"
 
 static BOOL isNightMode = NO;
-static NSDictionary<NSString *, UIColor *> *day;
-static NSDictionary<NSString *, UIColor *> *night;
 
 @implementation UIColor (MapsMeColor)
```

Comment:
Do we need this file/class at all?

Reply:
@MontakOleg Is this MapsMeColor still used somewhere?

Reply:
Ok, I see that there are some references. It would be great to rename it into OMColor in a separate PR.